src/plugins/intel_cpu/src/node.h

```diff
@@ -744,7 +717,7 @@ class Node {
 
     int fusingPort;
 
-    dnnl::engine engine;
+    const dnnl::engine engine;
```

Comment:
Why do we keep engine here given it is available inside the context? 
Looks worth to remove in separate PR

Reply:
OK, got it!